README.md

```diff
@@ -40,6 +40,9 @@ cd scylla-cdc-java
 mvn clean install
 ```
 
+By default, during the installation, a suite of Docker integration tests are ran. If you wish to disable them, pass a
```

Comment:
I hate being this kind of reviewer but... https://english.stackexchange.com/questions/560000/processes-are-run-vs-are-ran

Reply:
Fixed.